source/includes/_registry.md

```diff
@@ -0,0 +1,106 @@
+# Registry
+
+The registry is the low-level host functionality that provides the foundation
+for [SkyDB](#skydb). It allows getting and setting registry entries with a
+user's key and a data key (e.g. the name of an app).
+
+Registry entries contain data, the data key, and a revision number. The revision
+number is incremented by 1 every time the registry entry is changed. The latest
```

Comment:
it sounds like there is a check in the api that expects the revision + 1 - would it throw error if I set 12345678 even if it's larger than current revision?

Reply:
Not sure I understand your question, can you elaborate? It will definitely error if the desired new revision is the same or lower than the latest revision, but I *think* if it's larger it's actually OK